src/app/components/plugin-sidebar/plugin-sidebar.component.html

```diff
@@ -53,9 +49,41 @@ <h2 class="template-header">
             </mat-form-field>
         </div>
         <div [hidden]="activeArea != 'detail'">
-            Template Blah:
-
-            description...
+            <mat-divider></mat-divider>
+            <h3 class="sidebar-header">Template Details</h3>

```

Comment:
We may need to refactor the sidebar into multiple smaller components in the future